deployment/README.md

```diff
@@ -4,11 +4,36 @@
 
```

Comment:
Could we put a TOC at the top of this? It's now got five sections, navigation could be easier.

Reply:
Yeah that makes sense